aws_lambda_powertools/utilities/data_classes/api_gateway_proxy_event.py

```diff
@@ -33,6 +33,16 @@ def integration_latency(self) -> Optional[int]:
         """The authorizer latency in ms."""
         return self.get("integrationLatency")
 
+    def get_context(self) -> Optional[Dict[str, Any]]:
+        """Retrieve the authorization context details for Lambda.
+
+        Returns:
+        --------
+            Optional[Dict[str, Any]]
+            A dictionary containing Lambda authorization context details, or None if the information is not available.
+        """
+        return self
```

Comment:
```suggestion
    def get_context(self) -> Dict[str, Any]:
        """Retrieve the authorization context details injected by a Lambda Authorizer.

	Example
	--------
	
	```python
	ctx: dict = ...request_context.authorizer.get_context()
	
	tenant_id = ctx.get("tenant_id")
	```

        Returns:
        --------
        Dict[str, Any]
            A dictionary containing Lambda authorization context details.
        """
        return self
```